services/route_manager/src/main/java/com/futurewei/alcor/route/controller/RouterController.java

```diff
@@ -378,7 +455,7 @@ public RouteTableWebJson createNeutronSubnetRouteTable(@PathVariable String proj
                     "0.0.0.0/0", null, 100, null, "192.168.0.1");
             routes.add(defaultRoute);
 
-            routeTable = this.routerService.createNeutronSubnetRouteTable(projectid, subnetid, resource, routes);
+            routeTable = this.routerService.createSubnetRouteTable(projectid, subnetid, resource, routes);
```

Comment:
@DavidLiu506 We need to remove line#454~#456 defaultRoute setting.

Reply:
Removed.